lib/jet_ext/ecto/sti/parameterized.ex

```diff
@@ -0,0 +1,71 @@
+defmodule JetExt.Ecto.STI.Parameterized do
```

Comment:
```suggestion
defmodule JetExt.Ecto.STI.Type do
```
这里应该是一个类型，只不过其实现是一个 ParameterizedType，我觉得应该是这么个逻辑吧。

Reply:
这里想表达的是 STI 的 parameterized 的形式，就叫 Type 感觉是不是有点不准确？还有没其他的名字？

Reply:
我更激进一点，我觉得就叫 `JetExt.Ecto.STI` 就可以了